### PR TITLE
Some minor tweaks to help catch badness

### PR DESCRIPTION
**Pull request by Bidski**

### CODE REVIEW

File: module/platform/Webots/src/Webots.hpp [hunk 1]
```diff
@@ -130,17 +131,14 @@ namespace module::platform {
         std::vector<std::pair<NUClear::clock::time_point, Eigen::Affine3d>> Hwps;
 
         struct CameraContext {
-            Webots& reactor;
             std::string name;
             uint32_t id;
             message::input::Image::Lens lens;
             // Homogenous transform from platform (p) to camera where platform is the rigid body the camera is attached
```
Comment (KipHamiltons) [inline module/platform/Webots/src/Webots.hpp:137]: Hpc implies camera to platform but comment says opposite

**Review by KipHamiltons — COMMENTED** [review]
Seems fine on quick glance. Will review properly tomorrow

File: module/platform/Webots/src/Webots.hpp (hunk 1 above)
Comment (TrentHouliston) [reply module/platform/Webots/src/Webots.hpp:137]: And this is why you never put constants in comments when it can be avoided. Self documenting code good

**Review by tayloryoung6396 — DISMISSED** [review]
LGTM, move the folder to ensure config doesn't look in the cameras camera folder

**Comment by ysims** [comment]
This appears to reduce the amount of crashes (I didn't get any crashes when running it for a while).

**Review by KipHamiltons — APPROVED** [review]
Seems fine.
(I just pushed the formatting fix)

**Review by abihall — COMMENTED** [review]
I read it and it looks good

**Pull request merged**